dockers/docker-fpm-frr/supervisord.conf

```diff
@@ -0,0 +1,68 @@
+[supervisord]
```

Comment:
We may need look at "/usr/lib/frr/frr" service script and consolidate the changes here. like start option, sequence etc.  This made me to think what is the fundamental benefit for SONiC to control daemon level control instead of FRR service level control.

Reply:
i cannot find /usr/lib/frr/frr, I checked /usr/lib/frr/frrcommon.sh. then i found this vtysh -b to solve this integrated mode issue.